charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -1348,7 +1348,7 @@ spec:
                       type: object
                       properties:
                         fsGroup:
-                          type: string
+                          type: integer
```

Comment:
while you're at it, can you also add the `format: int64` under the `type: integer` for the `fsGroup` properties?

Reply:
@timflannagan1 Let's merge this to unbreak things, and can you add the format as a follow up PR?